db/migrate/20201215071122_add_status_to_report_table.rb

```diff
@@ -0,0 +1,6 @@
+class AddStatusToReportTable < ActiveRecord::Migration[6.0]
+  def change
+    add_column :reports, :status, :integer, default: Report::PENDING
```

Comment:
Just so you're aware, since the migration and code changes that use the new fields in the migration are in the same pull request, that new behavior will be broken when you deploy this PR until the migration is run.

Reply:
Hi @spinecone, I create a new PR for changes in schema and I also delete those migration files from this branch.